angeltest.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'Google\'s Search Functionality' : function (browser) {
+      browser
+        .url('https://xkcd.com')
+        .waitForElementVisible('body', 1000)
+        .pause(1000)
+        .assert.title('xkcd: Recombination And Reionization')
+        .useXpath() 
+        .click('//*[@id="middleContainer"]/ul[1]/li[3]')
+        .pause(5000)
```

Comment:
you did not check that the image has changed after clicking on the random button